pkg/routes/gacha-routes.go

```diff
@@ -10,6 +10,10 @@ var RegisterUserRoute = func(router *mux.Router) {
 	router.HandleFunc("/user/", controllers.GetUser).Methods("GET")
 	router.HandleFunc("/user/{userId}", controllers.GetUserById).Methods("GET")
 	router.HandleFunc("/user/{userId}", controllers.UpdateUser).Methods("PUT")
+	router.HandleFunc("/characters/", controllers.CreateCharacter).Methods("POST")
+	router.HandleFunc("/characters/list/", controllers.GetCharacters).Methods("GET")
```

Comment:
The name of the controller should be ListCharacters. Get is used for singular.
ref: https://cloud.google.com/apis/design/standard_methods

Reply:
Okay I will update accordingly.